### ISSUE TITLE
Progress / Roadmap?

### ISSUE THREAD

**Issue by RobertLowe** [blocknotes/activeadmin_materialize_theme#2]
Hey, hows it going?

Are you active with this?

Wondering if we can clean up some of the UI to behave like: https://github.com/vigetlabs/active_material

https://camo.githubusercontent.com/d1a2196060649a2df9db240351ab6c45cd617aaad70540fbfdfe15abea836aea/687474703a2f2f692e696d6775722e636f6d2f6b446b477a59652e706e67

Wishlist:

- Dropdown actions menu inside index
- Dropdown utility menu
- Filter/Inputs aligned horizontally

Generally the UI feels less spacious/more optimized in vigetlabs old implementation.

**Comment by blocknotes** [comment]
Hey @RobertLowe.
I use this theme for some projects, so I update it when needed.

I don't know that theme **vigetlabs/active_material**.
But if you like that design why changing plugin?

About the point you exposed:
- _"Dropdown actions menu inside index"_: the "Batch actions" menù is a dropdown, so I don't understand what you mean;
- _"Dropdown utility menu"_: in the menù bar right? I made a test and I see it's not working;
- _"Filter/Inputs aligned horizontally"_: I'll think about it, for me filters are fine as now BTW.
- _"Generally the UI feels less spacious/more optimized in vigetlabs old implementation."_: I don't know what Material library they use. In my theme I embedded Materialize framework and they have these spacings by default.
